doc/Configuration.rst

```diff
@@ -95,6 +95,15 @@ One common use case for this feature is to increase the maximum heap space avail
 
     Specifying 32GB of memory available to ImageJ ecosystem routines in the JVM.
 
+Using the SciJava REPL
+--------------------------------
```

Comment:
I'm wondering if this would be better suited to the new `Usage` page of the ReadTheDocs...

Reply:
Or the `Troubleshooting` page...

Reply:
Moved to the `Usage` page.